cmd/builder/command.go

```diff
@@ -15,6 +16,15 @@ type Command struct {
 	args    []string
 }
 
+// Executor holds available methods for command.
+type Executor interface {
```

Comment:
Although it is nice to have an verb as the interface name, I think in this case we could have the interface named `Command` only, and then the current struct `Command` becomes `DefaultCommand` - what do you think?